scripts/machine-launch-script.sh

```diff
@@ -231,6 +231,8 @@ set -o pipefail
         azure-identity==1.5.0 \
         azure-mgmt-compute \
         azure-mgmt-network \
+        fsspec \
+        s3fs \
```

Comment:
The requirements in this file a minimal and only serve to give the user enough of an environment to be able to clone the firesim repo and run `build_setup.sh`.  I see that other stuff has snuck in here since I last looked but are `fsspec` and `s3fs` needed before we have the manager?

Reply:
I think they may have been here in the original implementation of this PR because we hadn't yet moved the full requirements into conda-reqs.yml.

Reply:
Without these changes, `setup-manager` fails with this import path.
```
/opt/conda/envs/firesim/lib/python3.11/site-packages/paramiko/transport.py:178: CryptographyDeprecationWarning: Blowfish has been deprecated
  'class': algorithms.Blowfish,
Traceback (most recent call last):
  File "/home/centos/actions-runner-0/_work/firesim/firesim/./.github/scripts/install-firesim-pem.py", line 7, in <module>
    from common import manager_home_dir, manager_fsim_pem, set_fabric_firesim_pem
  File "/home/centos/actions-runner-0/_work/firesim/firesim/.github/scripts/common.py", line 6, in <module>
    from platform_lib import PlatformLib, AWSPlatformLib, AzurePlatformLib, Platform
  File "/home/centos/actions-runner-0/_work/firesim/firesim/.github/scripts/platform_lib.py", line 24, in <module>
    from awstools.awstools import get_instances_with_filter
  File "/home/centos/actions-runner-0/_work/firesim/firesim/deploy/awstools/awstools.py", line [33](https://github.com/firesim/firesim/actions/runs/4296629292/jobs/7488725823#step:3:34), in <module>
    from util.io import firesim_input
  File "/home/centos/actions-runner-0/_work/firesim/firesim/deploy/util/io.py", line 3, in <module>
    from fsspec.core import url_to_fs, open_local # type: ignore
    ```

(from https://github.com/firesim/firesim/actions/runs/4296629292/jobs/7488725823)

Reply:
Thanks.  I think I got Abe's attention to try and dedup the requirements a bit in #1445